packages/mongo/package.js

```diff
@@ -34,7 +34,8 @@ Package.onUse(function (api) {
     'diff-sequence',
     'mongo-id',
     'check',
-    'ecmascript'
+    'ecmascript',
+    'mongo-dev-server',
```

Comment:
Sure thing - I haven't had to build one myself, but as far as I understand things, you'll be looking to add your changes in `tools/upgraders.js`. The process seems to be pretty well documented in that file - you'll be adding a new upgrade name / section, and you can test things out by running `meteor admin run-upgrader myupgradername`. The `1.5-add-dynamic-import-package` upgrader section in that file might be a good one to copy / start from.